app/services/SectionsProvider.scala

```diff
@@ -54,6 +55,14 @@ class SectionsProvider @Inject()(protected val cacheConnector: DataCacheConnecto
   def taskRows(cache: Cache)(implicit messages: Messages): Seq[TaskRow] =
     mandatoryTaskRows(cache, messages) ++ dependentTaskRows(cache, messages)
 
+  def taskRowsForRenewal(cache: Cache)(implicit messages: Messages): Seq[TaskRow] =
+    taskRows(cache).filter(isTaskRowsForRenewal)
+
+  def taskRowsForRenewal(cacheId: String)(implicit ec: ExecutionContext, messages: Messages): Future[Seq[TaskRow]] =
+    taskRows(cacheId).map(_.filter(isTaskRowsForRenewal))
+
+  private def isTaskRowsForRenewal(taskRow: TaskRow): Boolean = taskRow.msgKey != BusinessMatching.messageKey
+
```

Comment:
yack at msgkey and messageKey variable naming, but guessing thats legacy and not something you've added here.